test_util/launch_configs/README.md

```diff
@@ -0,0 +1,46 @@
+
+### DC/OS Launch Deployment-specific Options
+* `ssh_user`: User that will be able to SSH into the cluster and run test. Cannot be set with `deployment_type: bare_cluster` as it will be inferred from the baked-in linux cluster template.
+* `ssh_private_key_path`: path to the private key that will allow the above user to SSH. Cannot be set if `key_helper` is true (see below). Must be set with `deployment_type: onprem` if `key_helper: false`
+* `key_helper`: can be `true` or `false`. Default: `false`. This feature will (for whichever provider you are using) create an SSH key pair for use with the deployment.
+  * If provider is aws, then this will create an AWS KeyPair with the same name as the `deployment_name` and attach it to the cloudformation or linux cluster. This KeyPair will be deleted with the cluster if dcos-launch is used for cluster deletion. This KeyPair will automatically be injected into the template parameters as KeyName
+  * If provider is azure, then one must provide a trusted public key to deploy the template. Thus, dcos-launch will make a private-public key and the parameters sshRSAPublicKey will be automatically populated
+  * GCE is not implemented yet, but it follows a similar paradigm to Azure
+* `zen_helper`: can be `true` or `false`. Default: `false`. dcos-launch is intended to provide turn-key deployments for any flavor we build. In the case of zen templates, AWS network resources must be specifically provisioned before deploying the zen template. With the zen helper, any of those missing prerequisites will be created and destroyed with the rest of the cluster. As in the key helper case, the generated temporary resources will be injected as the appropriate parameters in the zen template
+* `bare_cluster`: Used with `deployment_type: bare_cluster` or `onprem` while provider is not `bare_cluster`
+  * `os_name`: see [AWS AMI-OS map](../aws.py)
+  * `instance_type`: provider-specific string describing what type of hardware to request
+  * `instance_count`: number of instances to provision. This field is only used if `deployment_type` is not `onprem`. In that case, there are `num_masters`, `num_private_agents`, and `num_public_agents` fields which will implicitly determine `instance_count`
+* `dcos_parameters`: Used with `deployment_type: template` . Set as a nested key-value mapping describing which parameters to pass to the template pointed to with `template_url`. Not necessarily required as some templates have enough default settings to be launched without parameters
+* `dcos_config`: Used with `deployment_type: onprem`. This field will be the config to be passed to the onprem installer. There are a few difference between specifying a config here and one for the installer
```

Comment:
dcos_config should only include gen-native terms. use 'dcos_installer_config' for the installer specific bits